storybook/src/pages/amsterdam/home/Home.docs.mdx

```diff
@@ -1,12 +1,14 @@
 {/* @license CC0-1.0 */}
 
-import { Meta } from "@storybook/blocks";
+import { Meta, Primary } from "@storybook/blocks";
 import * as HomeStories from "./Home.stories.tsx";
 
 <Meta of={HomeStories} />
 
 # Home Page
 
+<Primary />
```

Comment:
You've added this on purpose? You don't mind the broken reflow?

Reply:
What reflow is broken?

Reply:
If you display a fullscreen story in the doc.mdx the breakpoints do not function properly since the adjust to the width of the screen and not the container for the story.